content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -8,25 +9,17 @@ tocDepth: 2
 
 <Admonition type="warning">
 
-Multiple database schema support is currently a very early, experimental preview feature. Introspection and applying your Prisma schema to your database with Prisma Migrate and `db push` are currently not supported. If you rely on them, please do not try this preview feature in the main branch of your project or in production.
-
-</Admonition>
-
-<Admonition type="warning">
-
 Multiple database schema support is currently only available with the PostgreSQL and SQL Server connectors.
 
 </Admonition>
 
-This guide explains how to include multiple database schemas in your Prisma schema, and how to query across multiple database schemas with Prisma Client.
-
-</TopBlock>
+Many database providers allow you to organize database tables into named groups. This can be done to make the logical structure of the data model easier to understand, or to avoid naming collisions between tables.
```

Comment:
I suggest adding second-person here in the second sentence to avoid passive voice

Reply:
Thanks, accepted Zach's suggestion below